packages/chart/src/components/EditorPanel/components/Panels/Panel.Annotate.tsx

```diff
@@ -0,0 +1,335 @@
+                    </label>
+                    <label>
+                      Horizontal Anchor
+                      <input
+                        type='checkbox'
+                        checked={config?.annotations[index].anchor.horizontal}
+                        onClick={e => {
+                          const updatedAnnotations = [...config?.annotations]
+                          updatedAnnotations[index].anchor.horizontal = e.target.checked
+                          updateConfig({
+                            ...config,
+                            annotations: updatedAnnotations
+                          })
+                        }}
+                      />
+                    </label> */}
+
+                    <label>
+                      Opacity
+                      <br />
+                      <input
+                        type='range'
+                        onClick={e => {
+                          const updatedAnnotations = [...config?.annotations]
+                          updatedAnnotations[index].opacity = e.target.value
+                          updateConfig({
+                            ...config,
+                            annotations: updatedAnnotations
+                          })
+                        }}
+                      />
+                    </label>
+
+                    <label>
+                      Edit Subject
+                      <input
+                        type='checkbox'
+                        checked={config?.annotations[index]?.edit?.subject}
+                        onClick={e => {
+                          const updatedAnnotations = [...config?.annotations]
+                          updatedAnnotations[index].edit.subject = e.target.checked
+                          updateConfig({
+                            ...config,
+                            annotations: updatedAnnotations
+                          })
+                        }}
+                      />
+                    </label>
+                    <label>
+                      Edit Label
+                      <input
+                        type='checkbox'
+                        checked={config?.annotations[index]?.edit?.label}
+                        onClick={e => {
+                          const updatedAnnotations = [...config?.annotations]
+                          updatedAnnotations[index].edit.label = e.target.checked
+                          updateConfig({
+                            ...config,
+                            annotations: updatedAnnotations
+                          })
+                        }}
+                      />
+                    </label>
+                    <label>
+                      Associated Series:
+                      <select
+                        onChange={e => {
+                          const updatedAnnotations = [...config?.annotations]
+                          updatedAnnotations[index].seriesKey = e.target.value
```

Comment:
I think this might be pointing to the same object and directly manipulating it because spread only does a shallow copy. _.cloneDeep will make a deep copy